tutorials/preprocessing/plot_35_artifact_correction_regression.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
+"""
+.. _tut-artifact-regression:
+
+Repairing artifacts with regression
+===================================
+
+This tutorial covers removal of artifacts using regression as in
+:footcite:`GrattonEtAl1983`.
+
+.. contents:: Page contents
+   :local:
+   :depth: 2
+
+We begin as always by importing the necessary Python modules and loading some
+data. In this case we use data from :ref:`brainstorm <tut-brainstorm-auditory>`
+because it has both EOG and ECG electrodes. We then crop it to 60 seconds,
+set some channel information, then process the data.
+
+Note that there are other corrections that are useful for this dataset that we
+will not apply here — see :ref:`tut-brainstorm-auditory` for more information.
```

Comment:
I'd prefer putting the `see :ref:`tut-brainstorm-auditory` for more information` part in parens instead of after a dash.